src/cbmc/cbmc_parse_options.cpp

```diff
@@ -602,6 +602,7 @@ int cbmc_parse_optionst::get_goto_program(
       }
 
       languaget *language=get_language_from_filename(filename);
+      language->get_language_options(cmdline);
```

Comment:
okk, I added it to `clobber...cpp` too, I also changed the exception message to include that maybe `get_language_options` was not called